packages/mds-compliance/api.ts

```diff
@@ -42,28 +43,9 @@ function api(app: express.Express): express.Express {
       // verify presence of provider_id
       if (!(req.path.includes('/health') || req.path === '/')) {
         if (res.locals.claims) {
-          const { provider_id } = res.locals.claims
-
-          /* istanbul ignore next */
-          if (!provider_id) {
-            await log.warn('Missing provider_id in', req.originalUrl)
-            return res.status(400).send({
-              result: 'missing provider_id'
-            })
+          if (!(await providerClaimMiddleware(req, res))) {
```

Comment:
Since this isn't standard Express middleware, could be rename this to `processProviderClaim`, `checkProviderClaim`, `getProviderClaim`, `validateProviderClaim` or something similar?